wgsl/index.bs

```diff
@@ -493,22 +538,32 @@ sampler_comparison
 ## Texture Functions ## {#texture-functions}
 
 <pre class='def'>
-vec4<type> texture_load(texture_ro, f32 | vec(2|3)<f32> coords, i32 level_of_detail)
+`vec4<type> texture_load(texture_ro, i32 | vec(2|3)<i32> coords, i32 level_of_detail)`
   %32 = OpImageFetch %v4float %31 %29 Lod %int_2
 
-TODO(dsinclair): Add `texture_write` method for texture_wo
+TODO(dsinclair): Texture load on a sampled texture
+
+TODO(dsinclair): Texture load on multi-sampled texture
+
+TODO(dsinclair): Add `texture_write` method for texture_wo with integral coords
 
-vec4<type> texture_sample(texture_sampled, sampler, f32 | vec(2|3)<f32> coords)
+TODO(dsinclair): Allow a small constant offset on the coordinate? May not be portable.
+
+`vec4<type> texture_sample(texture_sampled, sampler, f32 | vec(2|3)<f32> coords)`
```

Comment:
when sampling a cube array, the texture coordinate would be `vec4`

Reply:
Done